### ISSUE TITLE
headings for participants

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode-pull-request-github#5726]
@alexr00 the recording will make this clearer.

@jooyoungseo was surprised to find that PR view lacked headings that [github.com](http://github.com/) has (this is bc it’s a tree view)

We tried the PR description page and it has most of the headings, but it lacks headings for participants.

**Comment by alexr00** [comment]
As far as I can tell we do use `h3` for each comment header in a PR. I can't find the recording. Can you share again where the missing headers are?

**Comment by meganrogge** [comment]
@jooyoungseo could you pls share an example for where the headings are missing?

**Comment by alexr00** [comment]
Closing as no further information was provided.
